README.md

```diff
@@ -20,3 +20,77 @@ The following platforms are officially supported (tested):
 
 If you want to know how to build this project and contribute to it, please
 check out the [Contributing Guide](CONTRIBUTING.md).
+
+
+## Usage
+
+Please also refer to [examples](examples) for more detailed usage.
```

Comment:
The readme is included as-is as the [home page for the generated docs](https://github.com/frequenz-floss/frequenz-client-reporting-python/blob/v0.x.x/docs/index.md). This link won't work in the generated docs unless you also add some `examples/` section to the `docs/`.

Another alternative is to split the README into *includable* sections, and then in the generated docs home page, only include the sections that makes sense, like we do in the channels repo:
* https://raw.githubusercontent.com/frequenz-floss/frequenz-channels-python/v1.x.x/README.md
* https://raw.githubusercontent.com/frequenz-floss/frequenz-channels-python/v1.x.x/docs/index.md

You need to install the [`mkdocs-include-markdown-plugin`](https://pypi.org/project/mkdocs-include-markdown-plugin/) for this to work.

Reply:
>This link won't work in the generated docs unless you also add some examples/ section to the docs/.

Do you mean a subfolder? I have specified the full URL now.

Reply:
Yes and no, now the link will still work in the generated docs but IMHO is not ideal. It will point to the HEAD, which might be different than the version you are looking the docs for, for example, which could lead to confusion for users. You could get a "dynamic" link using macros, but if you do so, then the README will be screwed when read directly from GitHub.

What I mean is, again, ideally, is what I show above we did with the channels repo, where we include only some parts of the README in the generated docs.

A separate, but interesting discussion, is if we want to have an "examples" section in the generated docs, and have also kind of pick up the examples in `examples/` automatically or something. This could be done with some magic.

All that said, I'm OK with the current state as a temporary solution, just saying it has its flaws.

Reply:
Yes and no, now the link will still work in the generated docs but IMHO is not ideal. It will point to the HEAD, which might be different than the version you are looking the docs for, for example, which could lead to confusion for users. You could get a "dynamic" link using macros, but if you do so, then the README will be screwed when read directly from GitHub.

What I mean is, again, ideally, is what I show above we did with the channels repo, where we include only some parts of the README in the generated docs.

A separate, but interesting discussion, is if we want to have an "examples" section in the generated docs, and have also kind of pick up the examples in `examples/` automatically or something. This could be done with some magic.

All that said, I'm OK with the current state as a temporary solution, just saying it has its flaws.

Reply:
Ok, thank you for explaining. I would leave this for a future iteration.